packages/probing-eservice-operations/test/eservice.test.ts

```diff
@@ -485,13 +487,19 @@ describe("database test", async () => {
           audience: ["audience"],
         };
 
-        await eserviceService.saveEservice(eserviceId, versionId, payload);
+        const eserviceRecordId = await eserviceService.saveEservice(
+          eserviceId,
+          versionId,
+          payload
+        );
 
         const updatedEservice = await eservices.findOneBy({
           eserviceId,
           versionId,
         });
 
+        expect(eserviceRecordId).toBeTruthy();
+        expect(typeof eserviceRecordId).toBe("number");
```

Comment:
his is enforced by the type system, and cannot be properly verified in a unit test because it involves an external service
```suggestion
```